src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -470,27 +471,30 @@ export class SimpleSuggestWidget extends Disposable {
 				break;
 			case State.Open:
 				dom.hide(this._messageElement);
-				dom.show(this._listElement);
 				if (this._status) {
-					dom.show(this._status?.element);
+					dom.show(this._listElement, this._status?.element);
+				} else {
+					dom.show(this._listElement);
 				}
 				this._show();
 				break;
 			case State.Frozen:
 				dom.hide(this._messageElement);
-				dom.show(this._listElement);
 				if (this._status) {
-					dom.show(this._status?.element);
+					dom.show(this._listElement, this._status?.element);
```

Comment:
this is what the editor's one does https://github.com/microsoft/vscode/blob/478d09d2694b29da972b5038d0ce227eca3db4bd/src/vs/editor/contrib/suggest/browser/suggestWidget.ts#L493